server/src/controller/user.ts

```diff
@@ -112,11 +162,11 @@ export class UserController {
             // check if a user with the specified id exists
             // return a BAD REQUEST status code and error message
             ctx.status = 400;
-            ctx.body = "The user you are trying to update doesn't exist in the db";
+            ctx.body = { errors: {error: "The user you are trying to update doesn't exist in the db"}}
         } else if (await userRepository.findOne({ id: Not(Equal(userToBeUpdated.id)), email: userToBeUpdated.email })) {
             // return BAD REQUEST status code and email already exists error
             ctx.status = 400;
-            ctx.body = "The specified e-mail address already exists";
+            ctx.body = { errors: {error: "The specified e-mail address already exists"}}
         } else {
             // save the user contained in the PUT body
             const user = await userRepository.save(userToBeUpdated);
```

Comment:
not a new error but I noticed this while reviewing, this should probably be inside a try catch

Reply:
this endpoint is just a template for later, not in use now. Can't guarantee that the endpoint I write later will do this in the same way